sgp4/tests.py

```diff
@@ -165,6 +204,11 @@ class NewSatelliteObjectTests(TestCase, SatelliteObjectTests):
     def build_satrec(self, line1, line2):
         return Satrec.twoline2rv(line1, line2)
 
+    def build_satrec_sgp4init(self, satnum, jdsatepoch, bstar, ndot, nddot,
```

Comment:
Try expanding the name slightly: `build_satrec_and_call_sgp4init()`.

Reply:
Updated to be `build_satrec_from_sgp4init`